package.json

```diff
@@ -20,10 +20,12 @@
     "yeoman-generator"
   ],
   "dependencies": {
-    "lodash": "^4.7.11",
-    "yeoman-generator": "^0.19.0"
+    "lodash": "^4.17.19",
+    "yeoman-generator": "^4.11.0"
```

Comment:
Does `yeoman-generator` bump its node depedencies? If it does it is a breaking change and we should do a new major release.

Reply:
Yes a lot has changed between these as you can see in the changes that had to be made to the generators, also their dependencies changed, so it's breaking changes on yeoman side, wasn't sure how to reflect this in the generator itself.

Reply:
If we want to move to latest babel and jest in a follow up PR it might be worth combining the two in one major update?

Reply:
> how to reflect this in the generator itself.

`generator-babel-plugin` works for specific node versions. If by upgrading `yeoman-generator`, some legacy node versions turns unsupported, we should clearly say it and mark it as breaking changes.

> combining the two in one major update?

Sure we can release a new major after the overhauling is done. Thanks!